lib/main.js

```diff
@@ -229,8 +229,21 @@ ApigeeTool.listRoleUsers = function (opts, cb) {
   runCommand(cmd, opts, cb);
 };
 
+ApigeeTool.updateTargetServer = function (opts, cb) {
```

Comment:
Please move each new function closer to its friends. i.e. KVM functions together, Target Server functions together